README.rst

```diff
@@ -111,6 +111,7 @@ Europe
 * Spain (incl. Catalonia)
 * Slovenia
 * Switzerland
+* Turkey
```

Comment:
the `Vaud` area is a subdivision of the `Switzerland` calendar. To respect the alphabetical order, `Turkey` would rather be just above `United Kingdom`.